ops/pebble.py

```diff
@@ -12,7 +12,17 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-"""Client for the Pebble API (HTTP over Unix socket).
+"""Low-level Pebble API.
```

Comment:
I'm +1 on removing the "HTTP over Unix socket" here, but "Client for the Pebble API" (or just "Client for Pebble") feels more accurate than "Low-level Pebble API" to me.

Reply:
done, also updated index.rst